docs/ssss.md

```diff
@@ -1,3 +1,94 @@
 # Simperby Super Simple Summary
 
-TODO
+## DAO
+
+A DAO (Decentralized Autonomous Organization) is an organization that runs on a
+blockchain, governed by rules set in a contract, and operates autonomously
+without any government backing.
+
+For more information on DAOs, refer to [the introduction to DAO
+(WIP)](./dao.md).
+
+## How to Implement a DAO
+
+The most common way to implement a DAO is by **using a mainnet that supports
+smart contracts**. You can code your governance rules in a contract and deploy
+it on the mainnet, which will interact with the mainnet's ecosystem. However,
```

Comment:
What do you want to mean by mentioning "which will interact with the mainnet's ecosystem"?
Do you want to emphasize that the deployed code will 'serve as a DAO' by interacting with the mainnet ecosystem, or the deployed code has the capability to interact with the ecosystem (and thus powerful)?

Reply:
If there is no special reason to mention that,I would write
> One can implement a DAO by programming its governance rules into a smart contract and deplying it on a mainnet.

Reply:
I meant the latter, but I think the point 3 covers that. I fixed.